4-projects/business_unit_2/nonprod/example_floating_project.tf

```diff
@@ -16,21 +16,19 @@
 
 
 
-module "example_single_project" {
-  source = "../../modules/single_project"
-
+module "example_floating_project" {
+  source                      = "../../modules/single_project"
+  impersonate_service_account = var.terraform_service_account
   org_id                      = var.org_id
   billing_account             = var.billing_account
-  impersonate_service_account = var.terraform_service_account
+  folder_id                   = google_folder.business_unit_2.name
+  skip_gcloud_download        = var.skip_gcloud_download
   environment                 = "nonprod"
   env_code                    = "n"
-  skip_gcloud_download        = var.skip_gcloud_download
-
-  folder_id = var.parent_folder
 
   # Metadata
-  project_prefix    = "bu2-n-sample-single"
-  application_name  = "bu2-sample-application2"
+  project_prefix    = "bu2-p-sample-floating"
```

Comment:
this one should be `bu2-n-sample-floating` instead of  `bu2-p-sample-floating`

Reply:
The env code should be a variable to avoid this, no?

Reply:
This is in the context of the `nonprod` example so the only allowed value is `p`. Removing variables that had a value that cloud not be changed was a fix in previous PRs.